scripts/jest/config.base.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     '.*': require.resolve('./preprocessor.js'),
   },
   setupFiles: [require.resolve('./setupEnvironment.js')],
-  setupTestFrameworkScriptFile: require.resolve('./setupTests.js'),
+  setupFilesAfterEnv: [require.resolve('./setupTests.js')],
```

Comment:
We changed this setting to allow multiple files and name it something that makes more sense (shoutout to @ddruker)